pkg/test/testdata/cortex/config.yaml

```diff
@@ -99,3 +99,9 @@ ruler_storage:
   backend: filesystem
   filesystem:
     dir: "{{ .StorageDir }}/rules"
+
+limits:
+  metric_relabel_configs:
```

Comment:
is it possible to marshal the configuration used in cortex/config.go here for consistency?
```
MetricRelabelConfigs: []*relabel.Config{
				metrics.OpniInternalLabelFilter(),
			},
```

Reply:
Probably not, the actual production config is too different than the test one.